src/lib/hexdump.c

```diff
@@ -33,18 +33,18 @@ void hexdump(const void *memory, size_t length)
 		}
 
 		if ((all_zero < 2) && (all_one < 2)) {
-			printk(BIOS_DEBUG, "%p:", memory + i);
+			printk(BIOS_INFO, "%p:", memory + i);
 			for (j = 0; j < num_bytes; j++)
-				printk(BIOS_DEBUG, " %02x", line[j]);
+				printk(BIOS_INFO, " %02x", line[j]);
 			for (; j < 16; j++)
-				printk(BIOS_DEBUG, "   ");
-			printk(BIOS_DEBUG, "  ");
+				printk(BIOS_INFO, "   ");
+			printk(BIOS_INFO, "  ");
 			for (j = 0; j < num_bytes; j++)
-				printk(BIOS_DEBUG, "%c",
+				printk(BIOS_INFO, "%c",
 				       isprint(line[j]) ? line[j] : '.');
-			printk(BIOS_DEBUG, "\n");
+			printk(BIOS_INFO, "\n");
 		} else if ((all_zero == 2) || (all_one == 2)) {
-			printk(BIOS_DEBUG, "...\n");
```

Comment:
is this meant to be in this change?

Reply:
Certainly not.. Used it to lower the debug level

Reply:
Removed after rebasing.